### PR TITLE
Add `/help` and `/tutorial` commands and update bot activity

### PR DESCRIPTION
**Pull request by AlexanderHOtt**
- [x] `/help`
- [ ] `/tutorial`

### CODE REVIEW

**Comment by AlexanderHOtt** [comment]
image

**Comment by AlexanderHOtt** [comment]
I need some help writing the `/tutorial` command. I can style it and everything else, but I'm not sure how specific I should be.

**Review by yk — APPROVED** [review]

**Comment by yk** [comment]
I guess we can just make a video tutorial and then link to that

**Review by andreaskoepf — CHANGES_REQUESTED** [review]
Please show help for admin-commands only to bot-owner users.

File: discord-bot/bot/messages.py [hunk 1]
```diff
@@ -175,6 +179,61 @@ def confirm_ranking_response_message(content: str, items: list[str]) -> str:
...
+:small_orange_diamond: `rank_initial_prompts`: Rank some initial prompts
+:small_orange_diamond: `rank_prompter_replies`: Rank some prompter replies
+:small_orange_diamond: `rank_assistant_replies`: Rank some assistant replies
+
+To learn how to complete tasks, run `/tutorial`.
+"""
+    if can_manage_guild:
+        content += f"""\
+
+{_li("**`/settings log_channel <channel>`**")}
```
[the remark this answers is not in the document]
Comment (AlexanderHOtt) [reply discord-bot/bot/messages.py:209]: I agree that templates are probably better. 

If someone invites the bot to their server, they should be able to set the channel that the bot sends messages in. The log channel is a *server specific* configuration, so anyone who can manage the server should be able to use it

[the remark this answers is not in the document]
Comment (AlexanderHOtt) [reply discord-bot/bot/messages.py:209]: I can switch back to using templates in another PR, but I'll have to look up how to use them async. I think `aiofiles` might be the easiest way.

**Review by andreaskoepf — APPROVED** [review]

**Pull request merged**